src/util/include/util/raii/CLibrariesSPtr.h

```diff
@@ -67,23 +68,21 @@ class CLibrariesSPtr {
     constexpr static struct RefSink {
     } refsink = RefSink();
 
-    CLibrariesSPtr(T* p, Adopt = adopt): p(H::adopt(p)) {}
-    CLibrariesSPtr(T* p, Ref): p(safeRef(p)) {}
-    CLibrariesSPtr(T* p, RefSink): p(safeSinkRef(p)) {}
+    explicit CLibrariesSPtr(T* p, Adopt = adopt): p(safeAdopt(p)) {}
```

Comment:
We should make adopt explicit here. It's too easy to forget that the default is adopt instead of ref.